src/main/kotlin/study/racinggame/domain/Car.kt

```diff
@@ -1,13 +1,19 @@
 package study.racinggame.domain
 
 class Car(
-    var position: Int = 0
+    _position: Int = 1,
+    val name: String
 ) {
-    fun moveForward(randomNumber: Int) {
-        if (randomNumber > FORWARD_THRESHOLD) position += 1
+
+    init {
+        require(name.length <= 5) { "자동차 이름은 5자를 초과할 수 없습니다." }
     }
 
-    companion object {
-        private const val FORWARD_THRESHOLD = 3
+    private var position = _position
+
+    fun moveForward(moveAble: Boolean) {
+        if (moveAble) position += 1
     }
+
+    fun position() = position
```

Comment:
backing_property 사용 좋습니다!

```suggestion
    var position = _position
         private set

    fun moveForward(moveAble: Boolean) {
        if (moveAble) position += 1
    }

```

private set 을 사용하면 get에 대한 함수는 제거할 수 있지 않을까하네요~

Reply:
get을 제공하는 것 보다는 set을 막는 게 훨씬 좋네요.

이방법이 정말 코틀린스러운 좋은 해결법 같습니다